### DISCUSSION TITLE
Explicit punctuation

### DISCUSSION THREAD

**Discussion by 4voiceAG** [speechmatics/community#1] [Community help & support]
I'd like to test Speechmatics' performance for realtime dictation of documents (vs. subtitling or transcript of communication).
For this it's very important to be able to use explicit punctuation.
Unfortunately, I was not able to control the transcription to achieve this.
Setting ```permitted_punctuation``` to an empty array or ```sensitivity``` to 0 does not insert auto punctuation in the transcript, but unfortunately, a spoken "comma" didn't appear in the transcript either.

Is there a way to only turn off auto punctuation but have the engine recognize spoken punctuation words?

Addendum:
Further inspection of the recognized text revealed that capitalization due to unwanted auto punctuation is another key problem.
While I can try removing all the autopunctuation first and then replace the punctuation words by the correct symbols, this doesn't fix all the capitalized words produced by auto capitalization.

If there's no way to turn this off, then I'm afraid SpeechMatics is not suitable  for our use case.

**Comment by paulg-speechmatics** [comment]
Hi there! As you mention, setting `permitted_marks` in the `punctuation_overrides` section of the transcription config will remove all punctuation, but unfortunately the Speechmatics API does not allow the use of spoken punctuation.

With regards to the capitalization, is this occurring due to punctuation indicating the end of a sentence? Could you send a portion of the JSON output for review?

Thanks!

Paul
